src/components/Header.js

```diff
@@ -17,9 +18,13 @@ const Header = () => {
         }
       `}
       render={data => (
-        <div>
+
+        <div align="center">
```

Comment:
@josephsnyder can we please put these in a [Grid](https://material-ui.com/components/grid/)? Title and Logo in one column and Menu in the other.